package.json

```
@@ -42,10 +52,11 @@
     "build:requested-delivery-date": "npm --prefix feature-libs/requested-delivery-date run build:schematics && nx build requested-delivery-date --configuration production",
     "build:schematics": "npm --prefix projects/schematics run build",
     "build:setup": "nx build setup --configuration production",
-    "build:ssr": "env-cmd --no-override -e dev,b2c,$SPA_ENV nx run storefrontapp:server:production",
-    "build:ssr:opf": "env-cmd --no-override -e opf,b2c,$SPA_ENV nx run storefrontapp:server:production",
-    "build:ssr:ci": "env-cmd -e ci,b2c,$SPA_ENV nx run storefrontapp:server:production",
-    "build:ssr:local-http-backend": "env-cmd -e local-http,b2c,$SPA_ENV nx run storefrontapp:server:production",
+    "build:csr": "env-cmd --no-override -e dev,b2c,$SPA_ENV nx run storefrontapp:build:noSsr",
+    "build:ssr": "env-cmd --no-override -e dev,b2c,$SPA_ENV npm run build",
+    "build:ssr:opf": "env-cmd --no-override -e opf,b2c,$SPA_ENV npm run build",
+    "build:ssr:ci": "env-cmd -e ci,b2c,$SPA_ENV npm run build",
+    "build:ssr:local-http-backend": "env-cmd -e local-http,b2c,$SPA_ENV npm run build",
     "build:storefinder": "npm --prefix feature-libs/storefinder run build:schematics && nx build storefinder --configuration production",
```

[MEDIUM]
By running `storefrontapp:build:noSsr` I believe the build happens without production optimizations. In other words, `noSsr` configuration is not merged with the `production` configuration.

To address it, I propose to apply the analogical workaround that we applied for the prerendering (with a "proxy Nx target").

1. Let's change this line to:
```suggestion
    "build:csr": "env-cmd --no-override -e dev,b2c,$SPA_ENV nx run storefrontapp:build-csr-production",
```
2. And let's add the following target in the `project.json` (e.g. as a sibling under the target `"prerender-production"`):
```json
    "build-csr-production": {
      "executor": "@angular-devkit/architect:concat",
      "options": {
        "targets": [{ "target": "storefrontapp:build:production,noSsr" }]
      }
    },
```

---

Changes were added.

---

I really like your idea! It makes no sense to have `-production` suffix hardcoded in the name of the target.
TBH I tried it on Thursday, but couldn't get it to work (perhaps becasue of some typo in my config).
Now I used your above snippet and it works on my machine.

Would you mind then updating the targets `build-csr-production` and `prerender-production` according to your proposal? (so they have "configuration" parametrized).

---

Done. Please check.